cadasta/test/runtests.py

```diff
@@ -0,0 +1,51 @@
+#!/usr/bin/env python
+
+import argparse
+import os
+import sys
+import pytest
+from subprocess import Popen, DEVNULL
+
+
+if __name__ == '__main__':
+    # cadasta-test does not require Django
+    if 'DJANGO_SETTINGS_MODULE' in os.environ:
+        del os.environ['DJANGO_SETTINGS_MODULE']
```

Comment:
For some reason, this environment variable is automatically set when you log on to the dev VM. And since we have pytest-django installed, it checks if this environment variable is set and uses that specified Django settings to run pytest under. Otherwise it will just run the plain vanilla pytest.

If you have pytest-django installed and you run pytest in a directory where it can't find the Django settings, pytest fails. Since I am running pytest independent of Django, I have to unset this environment variable, if it is set, in order for the tests to run at all in the current shell session.